src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/ClientMock.swift

```diff
@@ -19,14 +19,18 @@
 import ExposureNotification
 
 final class ClientMock {
-	// MARK: Creating a Mock Client
-	init(submissionError: SubmissionError?) {
+	// MARK: - Creating a Mock Client.
+	init(submissionError: SubmissionError? = nil) {
 		self.submissionError = submissionError
 	}
 
-	// MARK: Properties
+	// MARK: - Properties.
 	let submissionError: SubmissionError?
+
+	// MARK: - Configurable Mock Callbacks.
```

Comment:
sometimes you have an empty line between MARKs and sometimes you don't. I'm fine with either, just please stick to one.